OWNERS_ALIASES

```diff
@@ -212,4 +190,24 @@ aliases:
     - jbeda
     - jdumars
     - mattfarina
+  provider-aws:
+    - d-nishi
+    - justinsb
+    - kris-nova
+  provider-azure:
+    - craiglpeters
+    - justaugustus
+    - feiskyer
+    - khenidak
+  provider-gcp:
+    - abgworrall
+  provider-ibmcloud:
+    - spzala
```

Comment:
For next time, changes to the leads list should probably be in a separate PR so the history is easier to follow for why this list may have changed :)